base/drivers/python_minimal/r/Dockerfile

```diff
@@ -39,10 +39,11 @@ RUN  ln -s /usr/lib/python3/dist-packages/apt_pkg.cpython-310-x86_64-linux-gnu.s
     gpg \
     gcc \
     build-essential \
+    sudo \
```

Comment:
Just to check, is this a system dependency for the r packages?

Reply:
It is needed for the R package `pak` and `raspy` ton function correctly when the container is running as a non privileged user.

Reply:
It's not strictly needed here because sudo is installed in the base image. I have included it so that if someone swaps out the FROM for another image they are more likely to end up with a functional build.

Reply:
thanks for confirming - wanted to make sure it wasn't included for some debugging or something